src/__tests__/SegmentedArc.spec.js

```diff
@@ -154,4 +154,10 @@ describe('SegmentedArc', () => {
     expect(Animated.timing).toHaveBeenCalledTimes(1);
     expect(Easing.out).toHaveBeenCalledWith(Easing.ease);
   });
+
+  it('sets the arc starting point', () => {
+    props.angleStart = 420;
```

Comment:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOHVpNmpsZXBlZ20xOW1sOXU4Mmp3dDB6c3cweGVzeGc4YWk1ZTJpNCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Bj5ILhCPm8EQ8/giphy.gif)

Reply:
`*Checks date*`, we missed an opportunity by 5 days